build/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+HOST_DIR=$(cd $(dirname "$0")/../; pwd)
+CONTAINER_DIR="/opt/rook.github.io"
+
+# install node dependencies through container
+echo "pulling npm dependencies..."
+docker run --rm --user 1000 -w $CONTAINER_DIR -v $HOST_DIR:$CONTAINER_DIR node:8-alpine npm install
```

Comment:
the --rm will remove the container after it exists so I'm not sure how `npm install` persist for the next run below. might be better to create a preprocess.sh that does npm install and runs node in a single script